code/modules/projectiles/updated_projectiles/gun_attachables.dm

```diff
@@ -684,6 +684,19 @@ obj/item/attachable/attack_hand(var/mob/user as mob)
 	scatter_unwielded_mod = -config.min_scatter_value
 
 
+/obj/item/attachable/stock/lasgun
+	name = "\improper M43 Sunfury lasgun stock"
+	desc = "The standard stock for the M43 Sunfury lasgun."
+	slot = "stock"
+	wield_delay_mod = null
```

Comment:
Add wield delay if you are giving a melee mod.

Reply:
It has a 1 second wield delay; this is higher even than the HPR after its wield delay nerf.